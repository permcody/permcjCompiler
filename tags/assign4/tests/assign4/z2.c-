//
// Randomly generated C- code
//
bool w(void)
{
    void w;
    bool x;
    int u;

    x=x=true;
    u=u=true;
    if (w = 0) {
	if (x + 7 - v)
	    v;
    }	
    {
	int x;
	x = u;
    }
    {
	bool x;
	x = u;
    }
}
