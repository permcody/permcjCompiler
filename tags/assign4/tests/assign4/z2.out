ERROR lineno(6): w cannot be of type void.
ERROR lineno(11): = requires things of the same type but lhs is int and rhs is bool.
ERROR lineno(12): Cannot use function w as a simple variable.
ERROR lineno(13): + requires things of type int but lhs is of type bool.
ERROR lineno(13): Symbol v is not defined.
ERROR lineno(14): Symbol v is not defined.
ERROR lineno(22): = requires things of the same type but lhs is bool and rhs is int.
Number of errors: 7
Number of warnings: 0
